UnoCheck/InteractiveSelector.cs

```diff
@@ -0,0 +1,183 @@
+using Spectre.Console;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace DotNetCheck
+{
+	/// <summary>
+	/// Provides interactive selection prompts for IDE and target platforms
+	/// </summary>
+	internal static class InteractiveSelector
+	{
+		/// <summary>
+		/// Determines if interactive prompts should be shown based on settings and environment
+		/// </summary>
+		public static bool ShouldShowInteractivePrompts(CheckSettings settings)
+		{
+			// Don't show in CI or non-interactive mode
+			if (settings.CI || settings.NonInteractive)
+				return false;
+
+			// Don't show if any relevant flags are already specified
+			if (!string.IsNullOrEmpty(settings.Ide))
+				return false;
+			
+			if (settings.TargetPlatforms?.Any() == true)
+				return false;
+			
+			if (settings.Frameworks?.Any() == true)
+				return false;
+			
+			if (settings.Skip?.Any() == true)
+				return false;
+
+			return true;
+		}
+
+		/// <summary>
+		/// Prompts user to select their IDE(s)
+		/// </summary>
+		/// <returns>
+		/// Array of IDE identifiers. If no IDEs selected, returns empty array and no IDE checks will be skipped.
+		/// </returns>
+		public static string[] PromptForIde()
+		{
+			var ideChoices = new List<string>();
+
+			// Show platform-appropriate IDEs
+			if (Util.IsWindows)
+			{
+				ideChoices.Add("Visual Studio");
+			}
+			
+			// Common IDEs across all platforms
+			ideChoices.Add("VS Code");
+			ideChoices.Add("Rider");
+			ideChoices.Add("Other / None");
+
+			var selectedIdes = AnsiConsole.Prompt(
+				new MultiSelectionPrompt<string>()
+					.Title("[bold blue]Which IDE(s) do you plan to use?[/]")
+					.PageSize(10)
+					.AddChoices(ideChoices)
+					.HighlightStyle(new Style(Color.Green)));
+
+			// If no IDEs selected, return empty array (no IDE-specific checks will be skipped)
+			if (!selectedIdes.Any())
+				return Array.Empty<string>();
+
+			// Map friendly names to internal identifiers
+			return selectedIdes.Select(ide => ide switch
+			{
+				"Visual Studio" => "vs",
+				"VS Code" => "vscode",
+				"Rider" => "rider",
+				"Other / None" => "other",
+				_ => "other"
+			}).ToArray();
```

Comment:
“Other / None” option, it currently maps to other, making “none” unreachable and maybe confusing here no ?